fem/sidredatacollection.cpp

```diff
@@ -385,38 +385,41 @@ createMeshBlueprintTopologies(bool hasBP, const std::string& mesh_name)
                             ? mesh->GetNE()
                             : mesh->GetNBE();
 
-   MFEM_VERIFY(num_elements > 0,
```

Comment:
Most of this changes to this function are due to the previous implementation's assumption that this function will only be called when there is at least one (boundary) element.